logstash-codec-line.gemspec

```
@@ -21,6 +21,8 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
+  s.add_runtime_dependency "logstash-mixin-ecs_compatibility_support", '~> 1.3', '< 2'
+  s.add_runtime_dependency "logstash-mixin-event_support", '< 2'
```

Similarly here, I would rather do a pessimistic spec to the _major.minor_ that has the features we need. Yes, they are technically the same, but this one positively declares what we need.

```suggestion
  s.add_runtime_dependency "logstash-mixin-event_support", '~> 1.0'
```